src/components/temperature-scale.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import Draggable from "react-draggable";
+import DragIcon from "../assets/drag.svg";
+import * as scaleKey from "../../sea-surface-temp-img/tempScaleKey.png";
```

Comment:
Note that this isn't in `src` and it wasn't really meant to be used I think. I think this scale is up to date, but I'm not entirely sure. It was just a screenshot that I added initially, so there's some reference.

I think it'd be neat if we could use this function:
https://github.com/concord-consortium/hurricane-model/blob/master/src/temperature-scale.js#L45-L48
to generate scale dynamically. That way, it'll always work, even if we change the color scheme one day. I guess it should be reasonably easy using canvas and 1-2px wide rectangles that have colors based on the output of this function. But if it's gonna take a long time, I'd skip it.

Reply:
thanks for the suggestion - I've tried a gradient background for the element since that worked pretty well as a quick solution.